espressostreamer/espresso_streamer.go

```diff
@@ -188,8 +244,8 @@ func (s *EspressoStreamer) parseEspressoTransaction(tx espressoTypes.Bytes) ([]*
 		log.Warn("failed to verify batch poster signature", "err", err)
 	}
 
-	if !success {
-		err = s.verifySignature(signature, userDataHashArr)
+	if !success && s.espressoTEEVerifier.GetLegacySgxVerifier() != nil {
```

Comment:
`GetLegacySgxVerifier` always returns nil so we are not testing the legacy anymore?

Reply:
Yeah that means we are running a completely new version where we don't need the verify the attestation like before

Reply:
Yeah that means we are running a completely new version where we don't need the verify the attestation like before